src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
```

Comment:
Create separate file for each block styles

Reply:
<img width="604" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/86789161/546f6212-11d7-482b-886b-874c7a16836a">